src/components/InlineSystemMessage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import Text from './Text';
+import * as Expensicons from './Icon/Expensicons';
+import colors from '../styles/colors';
+import Icon from './Icon';
+
+const propTypes = {
+    /** Error to display */
+    message: PropTypes.string.isRequired,
+};
+
+const InlineSystemMessage = (props) => {
+    if (props.message.length === 0) {
+        return null;
+    }
+    return (
+        <View style={[styles.flexRow, styles.alignItemsCenter]}>
+            <Icon src={Expensicons.Exclamation} fill={colors.red} />
```

Comment:
We should never reference `colors` directly in the UI – instead we should always reference [themes](https://github.com/Expensify/App/blob/2faf3d2b1f2db0bf7c85a559fb654ab46f3397c6/src/styles/themes/default.js). This is so that we can easily add darkmode (and potentially other themes?) later on.

Reply:
Created a quick follow-up PR https://github.com/Expensify/App/pull/9198

Reply:
Ah, nice spot Rory. Thanks